roles/k8s_best_practices_certsuite/tasks/pre-run.yml

```diff
@@ -201,8 +201,8 @@
     - name: Pull and tag certsuite image for the version under test
```

Comment:
To avoid issues, because we're handling simultaneously different versions, I'd save the org name in a variable, then we can use it depending on the case:

```
    # We need to pull the original image to tag it. Then, we remove the pulled
    # image
    - name: Pull and tag certsuite image for the version under test
      vars:
        _kbpc_repo_org_name: "{{ (kbpc_version is version('v5.2.3', '>=') or kbpc_version == 'HEAD') | ternary('redhat-best-practices-for-k8s', 'testnetworkfunction') }}"
      ansible.builtin.shell: |
        set -ex
        podman pull quay.io/{{ _kbpc_repo_org_name }}/{{ kbpc_image_name }}:{{ kbpc_version_image }}
        podman tag quay.io/{{ _kbpc_repo_org_name }}/{{ kbpc_image_name }}:{{ kbpc_version_image }} \
        {{ kbpc_certsuite_image }}
        podman image rm {{ kbpc_image_name }}:{{ kbpc_version_image }}
      register: _kbpc_pull_result
      retries: 2
      delay: 10
      until: _kbpc_pull_result.rc == 0
```